yb-voyager/src/utils/utils.go

```diff
@@ -337,6 +337,51 @@ func ContainsAnySubstringFromSlice(slice []string, s string) bool {
 	return false
 }
 
+func WaitForLineInLogFile(filePath string, message string) error {
+	// Wait for log file to be created
+	timeout := time.After(3 * time.Minute)
+	for {
+		_, err := os.Stat(filePath)
+		if err == nil {
+			break
+		}
+		select {
+		case <-timeout:
+			return fmt.Errorf("timeout while waiting for log file %q", filePath)
+		default:
+			time.Sleep(1 * time.Second)
+		}
+	}
+
+	file, err := os.Open(filePath)
+	if err != nil {
+		return fmt.Errorf("error opening file %s: %v", filePath, err)
+	}
+
+	defer file.Close()
+
+	for {
+		scanner := bufio.NewScanner(file)
```

Comment:
Hmm, I think it's okay that we read this every iteration and this is not very performant, since it's a one-time operation. However, pls get this part reviewed by Amit once.